apps/src/applab/ai/dropletConfig.js

```diff
@@ -0,0 +1,15 @@
+import * as api from '@cdo/apps/applab/api';
+
+export let aiConfig = {
+  blocks: [...aiBlocks]
```

Comment:
the answer to this is probably "yes," but does this work? i would think it would error because `aiBlocks` is defined below it, but maybe i don't understand something about how export statements are scoped?

Reply:
It does work, but I guess I'm not 100% sure why either.  I can swap the order if you think it could matter.

Reply:
that's okay, i think it's fine to leave as-is. if an error occurs or the blocks are missing in a different environment for whatever reason, it'll be clear what's going on!